### PR TITLE
Toggle mobile menu

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, kindly review my mobile-menu feature

### Changes made

- Add javascript to toggle mobile menu when hamburger icon is clicked
- Show mobile menu only on mobile screen sizes
- Close mobile menu on clicking 'X' close button

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where ESLint is not running. Kindly use the `linters.yml` from the [HTML-CSS-JS](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) folder of the linters config repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -10,7 +10,17 @@ menuIcon.addEventListener('click', () => {
...
+});
 
+mobileMenu.addEventListener('click', () => {
+  if (mobileMenu.style.display === 'block') {
+    mobileMenu.style.display = 'none';
+  }
+});
 
 const card = `
                     <div class="card">
```
Comment (zieeco) [inline script.js]: ### style.css file

- Hey, please, let's kindly adjust our styles to improve the design as much as we can below:
   
   - The height of the mobile menu should be 100% to take up the whole height of the screen. fix this. :pray: 
   - The position should be fixed so users don't get to see other parts of the webpage while scrolling. Please, fix this :pray: 
   - The width should be 100% to take the whole device width. please, fix this.




```
#mobile-menu {
  display: none;
  flex-direction: column;
  position: absolute;
  width: 95%;
  top: 0;
  bottom: 57px;
  background: #3c3a39;
  mix-blend-mode: normal;
  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
}
```

*Your design*
Screenshot from 2023-04-17 18-39-15

*your design*
Screenshot from 2023-04-17 18-41-58

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!

## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Mobile menu works correctly 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @egichuhis and @AleWaweru, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Mobile menu works correctly ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**